firmware/src/rdm_responder.c

```diff
@@ -0,0 +1,427 @@
+/*
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ * rdm_responder.c
+ * Copyright (C) 2015 Simon Newton
+ */
+#include "rdm_responder.h"
+
+#if HAVE_CONFIG_H
+// We're in the test environment
```

Comment:
Not really, if we've reached this point it means we're building with autotools, which is the test env.